src/_includes/layouts/learning-resources.njk

```diff
@@ -0,0 +1,62 @@
+---
+layout: nohero
+---
+<div class="w-full bg-white/50">
+<div class="eleventy md:grid md:grid-cols-[265px_1fr_auto] flex flex-col-reverse text-left pb-24 md:max-w-screen-xl m-auto">
+    {% include "./learning-resources-nav.njk" %}
+
+    <div class="max-w-screen-lg mx-auto px-8 pt-8">
+        <!-- Breadcrumbs - Desktop -->
+        <div class="hidden w-full mx-auto md:block">
+            <div class="font-medium border-b pb-1">
+                {% if version %}
+                v{{ version }}
+                {% endif %}
+                {% if page.url %}
+                    {{ page.url | handbookBreadcrumbs | safe }}
+                {% endif %}
+            </div>
+        </div>
+        <div class="w-full m-auto md:pt-6">
+            <div id="algolia-search" class="border rounded"></div>
+        </div>
+        <div class="w-full mx-auto flex flex-col md:flex-row">
+            <!-- Main Content -->
+            <div class="flex-grow order-last md:order-first">
+                <div class="mt-6 mb-4 prose prose-blue main-content">
+                    {{ content | rewriteHandbookLinks(page) | safe }}
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="border-l">
+        <!-- Breadcrumbs - Mobile -->
+        <div class="block px-8 pt-4 w-full mx-auto md:hidden">
+            <div class="font-medium border-b pb-1">
+                {% if version %}
+                v{{ version }}
+                {% endif %}
```

Comment:
What do you mean `version`?

Reply:
This was in the handbook template and I brought it in here when duplicated the file, I've removed the unused nunjucks blocks.